rest_framework_elasticsearch/es_pagination.py

```diff
@@ -28,3 +31,35 @@ def paginate_search(self, search, request, view=None):
         if self.count == 0 or self.offset > self.count:
             return []
         return list(search[self.offset:self.offset + self.limit])
+
+
+class ElasticPageNumberPagination(PageNumberPagination):
+
+    def paginate_search(self, search, request, view=None):
```

Comment:
Slight change of the https://github.com/encode/django-rest-framework/blob/master/rest_framework/pagination.py#L189